.github/workflows/minimal.yml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2023 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+name: Minimal workflow to test github action token
+
+on:
+  pull_request:
+    branches: master
+    types: [labeled]
+
+permissions: write-all
+
+jobs:
+  minimal_token_test:
+    name: minimal_token_test
+    runs-on: ubuntu-latest
+    if: |
+      (github.event.label.name == format('cp{0} beta', ':') || github.event.label.name == format('cp{0} stable', ':')) &&
+      (github.event.pull_request.merged == true)
+    steps:
+      - name: Checkout Flutter Repo
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
+        with:
+          repository: flutter/flutter
+          token: ${{ github.token }}
+          path: flutter
+          ref: master
+          fetch-depth: 0
+      - name: Write a random file
+        run: |
+          cd flutter
+          touch empty.json
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@153407881ec5c347639a548ade7d8ad1d6740e38
+        with:
+          path: flutter
+          commit-message: blah
+          committer: GitHub <noreply@github.com>
+          token: ${{ secrets.FLUTTERMIRRORINGBOT_TOKEN }}
```

Comment:
Please replace with `secrets.FLUTTERACTIONSBOT_CP_TOKEN`

Reply:
Sure, thanks for adding the token!